### PR TITLE
fix localnode null

### PR DESCRIPTION
**Pull request by bettybao1209**
close https://github.com/neo-project/neo-node/issues/762

### CODE REVIEW

File: neo-cli/CLI/MainService.Node.cs [hunk 1]
```diff
@@ -71,6 +71,7 @@ private void OnShowStateCommand()
                     uint headerHeight = NeoSystem.HeaderCache.Last?.Index ?? height;
 
                     Console.SetCursorPosition(0, 0);
+                    LocalNode ??= await NeoSystem.LocalNode.Ask<LocalNode>(new LocalNode.GetInstance());
```
Comment (shargon) [inline neo-cli/CLI/MainService.Node.cs:74]: How it could be null if it's asigned during `Start`?

Comment (ZhangTao1596) [reply neo-cli/CLI/MainService.Node.cs:74]: Because `Start` is `async`.

**Comment by cloud8little** [comment]
Test passed. When immediately input show state after starting neo, it shows height after 1~2 seconds.

**Review by cloud8little — APPROVED** [review]

**Comment by erikzhang** [comment]
Please wait for my real fix.

**Review by ZhangTao1596 — APPROVED** [review]

File: neo-cli/CLI/MainService.Node.cs (hunk 1 above)
Comment (ZhangTao1596) [reply neo-cli/CLI/MainService.Node.cs:74]: We should remove `LocalNode` ask in `Start`.

**Review by shargon — APPROVED** [review]

**Review by ZhangTao1596 — COMMENTED** [review]
It is a revert of #758. 
It can't fix https://github.com/neo-project/neo/issues/2380.
We can't trust global `LocalNode` as I said https://github.com/neo-project/neo-node/pull/763#discussion_r595070102.

**Comment by superboyiii** [comment]
Yes, if make it like this, we will remeet the issue of https://github.com/neo-project/neo/issues/2380.
image

**Comment by erikzhang** [comment]
Running two different nodes on a single server with the same port is not a normal usage. And the second node should throw exception and be terminated. I think the fix of neo-project/neo#2380 should be in the core.

**Pull request merged**